app/models/concerns/votes_counter.rb

```diff
@@ -30,4 +30,9 @@ def votes_alternative
       sum + vote.value
     end
   end
+
+  def vote_by(user)
+    vote = votes.select { |vote| vote.user == user }.first
```

Comment:
I think i've found the method we were looking for: `find`
You could now do
```ruby
  vote = votes.find{|vote| vote.user ==  user }
```
even better, if we don't want to add the `eager_load`, we could do: 
```ruby
  vote = votes.find{|vote| vote.user_id ==  user.id }
```